tools/nxfuse/README.txt

```diff
@@ -113,4 +113,10 @@ root using "sudo apt-get install libfuse-dev".
 
 4) File Systems -> Memory Technology Device (MTD) Support -> SMART Device options ->  Enable "Enable Sector CRC error detection" (=y)
 
-5) Adjust blkcount in os/tools/mksmartfsimg.sh. blkcount multiplied by blksize should be equal to SMARTFS partition size.
+5) Adjust blkcount in TizenRT/tools/nxfuse/mksmartfsimg.sh. blkcount multiplied by blksize should be equal to SMARTFS partition size.
```

Comment:
1. How about using markdown syntax to show it on github?
2. This file is difficult to see. If I am an user, I don't have any interesting about nxfuse command. I think I just want to know how to make and how to use.
 Let's make it like below
```
# How to use nxfuse
The nxfuse is ....

## Contents

> [How to make smartfs image on TizenRT](#how-to-make-smartfs-image-on-tizenrt)  
> [APPENDIX](#appendix)  

## How to make smartfs image on TizenRT
### Configurations
enable configurations as followings
1) Build Setup -> Binary Output Formats -> [*] Make smartfs binary image

2) Board Selection -> Adjust smartfs partition size

3) File Systems -> Memory Technology Device (MTD) Support -> SMART Device options ->  set "SMART Device sector size" = (512 bytes)

4) File Systems -> Memory Technology Device (MTD) Support -> SMART Device options ->  [*] Enable Sector CRC error detection
### SmartFS Partition size
Adjust blkcount in TizenRT/tools/nxfuse/mksmartfsimg.sh. blkcount multiplied by blksize should be equal to SMARTFS partition size.

### Content
Place the user files to be loaded, in path "TizenRT/tools/fs/contents-nxfuse/"CONFIG_ARCH_BOARD"/base-files/" (CONFIG_ARCH_BOARD--> refer os/.config)
   For Ex: If "CONFIG_ARCH_BOARD=imxrt1020-evk", then the base-files path would be "TizenRT/tools/fs/contents-nxfuse/imxrt1020-evk/base-files/"

### Command to make smartfs image
at os folder ...
./dbuild.sh smartfs

## APPENDIX
(previous generic nxfuse description)
```